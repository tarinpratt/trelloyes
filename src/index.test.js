import React from 'react';
import ReactDOM from 'react-dom';
import List from './list';
import Card from './card';

it('renders without crashing', () =>{
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('renders without crashing', () =>{
    const section = document.createElement('section');
    ReactDOM.render(<List />, section);
    ReactDOM.unmountComponentAtNode(section);
});