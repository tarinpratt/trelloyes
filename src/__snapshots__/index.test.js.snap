// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<section
  className="List"
>
  <header />
  <div
    className="List-cards"
  />
  <button
    className="List-add-button"
    type="button"
  >
    + Add Random Card
  </button>
</section>
`;

exports[`renders the UI as expected 2`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;
